src/screens/TopicPageScreen/elements/NavHeader.js

```diff
@@ -272,15 +264,15 @@ const styles = StyleSheet.create({
     zIndex: 80,
     overflow: 'hidden'
   }),
-  navContainer: (isHeaderHide, top) => ({
+  navContainer: (isHeaderHide, headerHeight) => ({
     flexDirection: 'row',
-    height: isHeaderHide
-      ? dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT2
-      : dimen.size.TOPIC_FEED_NAVIGATION_HEIGHT,
     justifyContent: 'space-between',
-    alignItems: 'flex-start',
-    backgroundColor: 'white',
-    zIndex: 10
+    alignItems: 'center',
+    zIndex: 10,
+    height: headerHeight,
+    backgroundColor: COLORS.almostBlack,
+    borderBottomWidth: isHeaderHide ? 1 : 0,
+    borderBottomColor: COLORS.gray310
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The changes in this section are mainly about style adjustments. The new code seems to be more consistent and readable. However, there's a change in the `alignItems` property from 'flex-start' to 'center'. This might affect the layout depending on the content inside. Make sure this change doesn't break the layout.


<!-- This is an auto-generated comment by OSS CodeRabbit -->